### ISSUE TITLE
Scrolling very slow

### ISSUE THREAD

**Issue by MrZeroo00** [zegervdv/homebrew-zathura#37]
Hello,

I'm not sure this is the right place to get help but I figured fellow zathura macOS users could help me.
I installed zathura using this formulae, it works but I have a problem with the scrolling.
It's very slow... I tried changing scroll-step to various values, enabling smooth-scroll and changing the backend (poppler and mupdf) but it doesn't seem to change anything.
Here's a video where we can see the issue: https://dsh.re/b8daf
I have a Macbook Pro 2017 (3,1 GHz Intel Core i5 and 16 GB of RAM) and I am using zathura in full screen on a 4K screen. On smaller screens it's a bit faster.
Did anyone have this issue?

Thanks

**Comment by hcf-n** [comment]
After a fresh install I have the same issue, but only in fullscreen. I'm on OS X 10.15.5

**Comment by SylvanFranklin** [comment]
Is this still an issue for you? I haven't been able to reproduce it

**Comment by JayanAXHF** [comment]
> Is this still an issue for you? I haven't been able to reproduce it

Yeah, on the contrary, Zathura scrolling is _way too fast_ for me.

**Comment by badrbouslikhin** [comment]
I moved to sioyek since then.
